gui/src/ts/Hrefs.ts

```diff
@@ -75,7 +75,7 @@ export class Hrefs {
         );
       },
 
-      paths: kgPrefix + "path",
+      // paths: kgPrefix + "path",
```

Comment:
Just nuke it since the pages have disappeared.

Reply:
Removed